src/scaffold.ts

```diff
@@ -56,7 +56,7 @@ export const scaffoldProjectRequest = async (item?: KafkaCluster | KafkaTopic) =
       (template) => template.spec!.display_name === pickedItem?.label,
     );
   } catch (err) {
-    logError(err, "template listing", {}, true);
+    logError(err, "template listing", { extra: {} });
```

Comment:
For these situations where we wanted to forward a message to Sentry but didn't have any data/tags, what do you think would be most helpful to send as part of `ScopeContext`? The current function / class+method name maybe?

Reply:
makes sense... something like
```
      logError(e, `"${selection}" button callback`, {
        extra: { function_name: "showNotifcationWithButtons" },
      });
```

Reply:
Yep, just camel case 😄